GoogleAddressAutocompletionPlugin/SNCGoogleMapsAPI.m

```diff
@@ -15,6 +16,9 @@
 static NSString *addressAutocomplete = @"https://maps.googleapis.com/maps/api/place/autocomplete/json";
 static NSString *placeDetails = @"https://maps.googleapis.com/maps/api/place/details/json";
 static NSString *textSearchPlace = @"https://maps.googleapis.com/maps/api/place/textsearch/json";
+static NSString *placePhoto = @"https://maps.googleapis.com/maps/api/place/photo";
+
+static CGFloat defaultImageMaxWidth = 1000;
```

Comment:
Maybe using standard values like 1024 would be better

Reply:
Done.